### ISSUE TITLE
Add serialization of debugger state (global and per-target).

### ISSUE THREAD

**Issue by benvanik** [benvanik/xenia#428]
Should save window positions/splitters/etc, as well as view options (ppc/x64/etc) globally. Target-specific stuff like breakpoints and map files should be saved per-target (maybe side-by-side? or in some maps/ path?).

**Comment by DrChat** [comment]
We can use this for save states as well.

**Comment by benvanik** [comment]
Yeah for sure! there's a --break_on_start flag, would be nice to have a --load_savestate or something as well - could then launch dumps and step through them!
